Mathlib/Tactic/GeneralizeProofs.lean

```diff
@@ -194,7 +194,7 @@ def mkLambdaFVarsUsedOnly (fvars : Array Expr) (e : Expr) : MetaM (Array Expr ×
   for i' in [0:fvars.size] do
     let i := fvars.size - i' - 1
     let fvar := fvars[i]!
-    e ← mkLambdaFVars #[fvar] e
+    e ← mkLambdaFVars #[fvar] e (usedOnly := false) (usedLetOnly := false)
```

Comment:
Oops, the fact that `mkLambdaFVars` doesn't include unused `let`s by default has surprised me too often.

Another fix would be to have line 206 be `| _ => pure ()`